### DISCUSSION TITLE
How to get multiple Atropos on the same page?

### DISCUSSION THREAD

**Discussion by BrandonCronk** [nolimits4web/atropos#8] [Q&A] [answered]
How do I get multiple cards on the same page to work with the same class? I noticed that your homepage example has a class called movie-box and game-box for multiple examples on the same page. When I initialize Atropos, I only get the effect on the first instance of that class.

**Comment by nolimits4web** [comment, marked as answer]
You need to initialize Atropos for every element, e.g.:
```js
document.querySelectorAll('.my-atropos').forEach((element) => {
  Atropos({
    // pass unique element here
    el: element,
    // rest of parameters
  });
})
